app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -122,10 +129,10 @@ class MainViewModel : ViewModel() {
         }
     }
 
-    fun updateUserInfo() = viewModelScope.launch(Dispatchers.IO) {
+    fun updateUserInfo(context: Context) = viewModelScope.launch(Dispatchers.IO) {
```

Comment:
This `ViewModel` can already have a context by changing its super class to `AndroidViewModel`.